sign_rpms

```diff
@@ -2,17 +2,20 @@
 
 . settings
 
-copy_gpg_key
+gpg_pass=$(mktemp)
+trap 'shred --remove "${gpg_pass}"' EXIT
+chmod 0700 "$gpg_pass"
+show_gpg_password > "$gpg_pass"
+
+SIGN_FLAGS="--define '_gpg_sign_cmd_extra_args --pinentry-mode loopback --passphrase-file ${gpg_pass}' --define '_gpg_path $KEYDIR' --define '_gpg_name $SIGNER'"
```

Comment:
TODO: this should probably become an array

Reply:
What are your thoughts on inserting TODO items as comments in the source files in the case of these scripts?  I think this would help identify areas that need improvement in reviewing the code.

Reply:
I think that we should apply shellcheck on this repository, which highlighted it locally for me. I'll take a quick look at fixing it.